reservations/schema.py

```diff
@@ -58,6 +69,7 @@ class ReservationInput(graphene.InputObjectType):
 class CreateReservation(graphene.Mutation):
     class Arguments:
         reservation = ReservationInput()
+        berth_switch = BerthSwitchInput(required=False)
```

Comment:
`required=False` is deffault behavior, so you could omit that. might be a good idea to add `required=True` to `ReservationInput()` though